rocAL/rocAL_pybind/example/new_api/PYTHON_UNITTEST_TEST_FILE.sh

```diff
@@ -97,36 +97,36 @@ rocAL_api_python_unittest=1
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name crop_mirror_normalize --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name nop --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name centre_crop --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name color_temp --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name copy --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name rotate_fisheye_fog --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name resize_brightness_jitter --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name vignetter_blur --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
-    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name snow --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 2 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
+    python$ver rocAL_api_python_unittest.py --image-dataset-path $data_dir --augmentation-name resize --batch-size $batch_size --display --NHWC --local-rank 0 --world-size $gpus_per_node --num-threads 1 --num-epochs 1 --$backend_arg 2>&1 | tee -a run.rocAL_api_log.${CURRENTDATE}
```

Comment:
I think, it is better to keep it as 2 epochs so we can get more test coverage. Is there any reason why it is needed?

Reply:
No specific reason Rajy.

I will change it back to 2 epochs.